src/app/profile/addActivityModal/addActivityForm/AddActivityForm.tsx

```diff
@@ -38,6 +47,14 @@ export const AddActivityForm = ({ onSubmit }: AddActivityFormProps) => {
       value: discipline,
     }));
 
+  useEffect(() => {
+    if (isFromSubmittedSuccessfully) {
+      showSnackbar(formatMessage({ id: 'addActivity.success' }));
```

Comment:
Wiesz co, ta część powinna być wrzucona gdzieś wyżej. Moim zdaniem w miejscu, gdzie jest wywoływany modal jako child.
Single Principle Responsibility, pamiętasz? Jeden twór ma robić jedną rzecz. Form ma zbierać dane, i tylko to. Wyświetlanie sukcesów powinno się zawierać w bardziej ogólnych komponentach